base_multi_company/models/res_company_assignment.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models, tools
+
+
+class ResCompanyAssignment(models.Model):
+
+    _name = 'res.company.assignment'
+    _auto = False
+
+    name = fields.Char()
+
+    @api.model_cr
+    def init(self):
+        tools.drop_view_if_exists(self.env.cr, self._table)
```

Comment:
IMO if you provides the search method the company_id will be usable... and the compute should be replaced as follow
```python
@api.multi
    @api.depends('company_ids')
    def _compute_company_id(self):
       user_companies = self.env['res.company'].search([])
        for record in self:
            record.company_id = False
            for company in record.company_ids:
                if company_id.id in user_companies.ids:
                    record.company_id = company_id
                    break
```